service/src/main/java/bio/terra/profile/app/Main.java

```diff
@@ -34,6 +35,7 @@
 @ConfigurationPropertiesScan(basePackages = {"bio.terra.profile"})
 @EnableRetry
 @EnableTransactionManagement
+@ServletComponentScan
```

Comment:
What is this?

Reply:
It tells Spring to look for Servlet components. The `@WebFilter` component isn't found by the normal `ComponentScan`, so this is required for Spring to find the new filter. Without it, BPM won't add the `Cache-Control` header. 

There's a little more info here - https://www.baeldung.com/spring-servletcomponentscan#using-servletcomponentscan-in-spring-boot